skafka/src/test/scala/com/evolutiongaming/skafka/consumer/ExplodingConsumer.scala

```diff
@@ -0,0 +1,130 @@
+package com.evolutiongaming.skafka.consumer
+
+import java.lang.{Long => LongJ}
+import java.time.{Duration => DurationJ}
+import java.util.concurrent.{TimeUnit => TimeUnitJ}
+import java.util.regex.Pattern
+import java.util.{Collection => CollectionJ, List => ListJ, Map => MapJ, Set => SetJ}
+
+import org.apache.kafka.clients.consumer
+import org.apache.kafka.clients.consumer.{
+  ConsumerRebalanceListener,
+  OffsetAndMetadata,
+  OffsetCommitCallback,
+  Consumer => ConsumerJ
+}
+import org.apache.kafka.common.{Metric, MetricName, PartitionInfo, TopicPartition => TopicPartitionJ}
+
+/**
+  * It is intentional to have all methods as `???` (throws NotImplementedError)
+  *
+  * It is used to verify the only expected interaction in corresponding tests
+  * by implementing the only expected method to be called in test
+  */
+class ExplodingConsumer extends ConsumerJ[String, String] {
+  def assignment(): SetJ[TopicPartitionJ] = ???
```

Comment:
please add custom exception in order to remove `???`

Reply:
fixed in 8f3dad10